frontend/test/metabase/scenarios/question/notebook.cy.spec.js

```diff
@@ -255,10 +255,9 @@ describe("scenarios > question > notebook", () => {
     });
   });
 
-  describe("nested", () => {
+  describe.only("nested", () => {
```

Comment:
The `.only` change is related or unintentional?

Reply:
Great catch, @ariya . Thank you.
That was unintentional - leftover from a local testing.